package.js

```diff
@@ -78,7 +78,6 @@ Package.onUse(function(api) {
     // 3rd Party
     "assets/js/third_party/embedly.min.js",
     "assets/js/third_party/font_awesome.min.js",
-    "assets/js/third_party/image_aviary.min.js",
```

Comment:
why you have removed image_aviary???

Reply:
It is not there in dist files shared by stefan. So was giving compilation error.

Reply:
dist was just for testing.  cant change the framework code because of that.

Reply:
why its not part of our dist. Check.?